### ISSUE TITLE
[Feature request] Sort By inside Playlist

### ISSUE THREAD

**Issue by andiandi13** [vanilla-music/vanilla#372]
I really think you should add the same sort by option that we see in the songs but inside a playlist, it is annoying in a favorites playlist to search songs one by one, it would be far easier to sort by date, artist or name for example.

I apologize for my english

**Comment by andiandi13** [comment]
(The Fav playlist is only an example)

**Comment by un-logic** [comment]
@andiandi13 
Do you mean playlist editor (one that has to buttons on top) must have an ability to automatically order songs albeit manual dragging of items?
That would have been a really nice feature. And it can reuse existing sorting methods:
 - by date added (to playlist, so one could undo changes easily)
 - by album, track number
 - by artist, album, track number
 - by artist, album, title
 - by title
 - by genre
 - by play count
 - by year (realised)
 - by artist, year
 - by whatever else you wish

**Comment by andiandi13** [comment]
@un-logic Exactly, when you tap the arrow and there is Edit and Delete at the top.
And yes there already are sorting methods so I don't think its hard to implement. 
Because the manual reordering is not very effective on this matter :p

**Comment by andiandi13** [comment]
@adrian-bl What do you think ?

**Comment by vespadj** [comment]
File (=Folder) playback isn't sorted by (path/)filename.
Please, consider to implement this or give a chance.

**Comment by un-logic** [comment]
@vespadj your request relates to issue #61 I guess. This one is about Playlist tab rather than Files tab.

**Comment by adrian-bl** [comment]
I'll keep an eye on this while working in #460: having an SQL view for playlists which is similar to what we have for tracks would simplify this
